### PR TITLE
Roc 1814 legal integration first test

### PR DESCRIPTION
**Pull request by vianvi**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ services:
       image: docker.artifactory.reform.hmcts.net/cmc/legal-integration-tests:${LEGAL_INTEGRATION_TESTS_VERSION:-latest}
       environment:
         - WEB_DRIVER_HOST=remote-webdriver
-        - URL=https://legal-frontend:4000
+        - URL=https://www-local-legal-frontend.moneyclaim.reform.hmcts.net:4000
```
Comment (dharmendrak) [inline docker-compose.yml:16]: url is dev-legal.moneyclaim.reform.hmcts.net

File: docker-compose.yml [hunk 2]
```diff
@@ -52,8 +52,12 @@ services:
           condition: service_healthy
         service-auth-provider-api:
           condition: service_started
+      networks:
+        default:
+          aliases:
+            - www-local-legal-frontend.moneyclaim.reform.hmcts.net
```
Comment (dharmendrak) [inline docker-compose.yml:58]: here as well dev-legal.moneyclaim.reform.hmcts.net

File: tests/home/steps/user.js [hunk 3]
```diff
@@ -1,17 +1,28 @@
 'use strict'
 
-const user = require('../../../test-data').user
+const user = require('../../../test-data').user;
```
Comment (dharmendrak) [inline tests/home/steps/user.js]: ; remove this

File: tests/home/steps/user.js [hunk 4]
```diff
@@ -1,17 +1,28 @@
...
 
-let loginPage, startClaimPage, taskListPage
+let loginPage, startClaimPage,enterYourDetailsPage;
```
Comment (dharmendrak) [inline tests/home/steps/user.js]: remove ; and add space after ,

**Review by dharmendrak — CHANGES_REQUESTED** [review]
few comments, otherwise it's all good

File: docker-compose.yml (hunk 2 above)
Comment (vianvi) [reply docker-compose.yml:58]: this is only required for local

File: docker-compose.yml (hunk 1 above)
Comment (vianvi) [reply docker-compose.yml:16]: this is only required for local

File: docker-compose.yml (hunk 2 above)
Comment (dharmendrak) [reply docker-compose.yml:58]: ignore it

**Review by dharmendrak — DISMISSED** [review]
LGTM

**Review by dharmendrak — APPROVED** [review]

**Pull request merged**